### PR TITLE
Corrected IsWorking NullReferenceException

### PR DESCRIPTION
**Pull request by Duke02**
Also added test to ensure that it works.

### CODE REVIEW

**Comment by Duke02** [comment]
I should add that this attempts to fix #14

**Review by vladjerca — APPROVED** [review]

**Review by vladjerca — CHANGES_REQUESTED** [review]
@Duke02 Seems that the tests are failing? Seems to be related to the imported namespaces.

They fail on both `Linux` and `OSX`, same error.

More details here: https://travis-ci.org/vladjerca/FFMpegCore/builds/545786272?utm_source=github_status&utm_medium=notification

LE:

After cloning your `fork`, I found what's missing, just add the following imports to the `test file`. (strange that you didn't get a build error locally 😕)

```csharp
using System.Collections.Generic;
using System.Drawing;
using System.Drawing.Imaging;
using System.IO;
using System.Linq;
```

Also, after importing the required namespaces, I noticed all of the other tests fail due to the modified code.

File: FFMpegCore/FFMPEG/FFBase.cs [hunk 1]
```diff
@@ -25,20 +25,14 @@ protected FFBase()
...
-                }
-                catch
+                if (Process != null)
                 {
-                    processHasExited = true;
+                    return !Process.HasExited && Process.GetProcesses().Any(x => x.Id == Process.Id);
                 }
 
-                return !processHasExited && Process.GetProcesses().Any(x => x.Id == Process.Id);
+                return true;
```
Comment (vladjerca) [inline FFMpegCore/FFMPEG/FFBase.cs:35]: This should implicitly return `false`. This is why all of the tests currently fail 😬

**Review by vladjerca — CHANGES_REQUESTED** [review]

**Comment by vladjerca** [comment]
@Duke02 Are you still working on this issue?

**Comment by Duke02** [comment]
No I am not. Sorry.

**Pull request closed without merging**